application/controllers/UploaderController.php

```diff
@@ -237,11 +237,43 @@ public function run($actionID)
                 echo ls_json_encode($return);
                 Yii::app()->end();
             }
-
+            // event to allow updating some part
+            $event = new PluginEvent('beforeMovePublicFileUploaded');
+            $event->set('surveyId', $surveyid);
+            $event->set('responseId', Yii::app()->session['survey_' . $surveyid]['srid']); // NULL if not exist
+            $event->set('qid', $oQuestion->qid);
+            $event->set('success', true);
+            $event->set('filename', $filename);
+            $event->set('size', $size);
+            $event->set('tmp_name', $_FILES['uploadfile']['tmp_name']);
+            App()->getPluginManager()->dispatchEvent($event);
+            $success = $event->get('success', true);
```

Comment:
Message in case of success is not later set in the output

Reply:
There are no messages in case of success.

I don't understand

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/3991/files/f97eba89064b5abc4ab5fcc7e05cced6c4cebdfb#diff-da3ae6cdc4a22c11c24a0ed743eb4bc1a190312eda624342eff57cc1e27883eeR313

Here a success message is shown, but the message can´t be set from the plugin.
That was what I was referring.

Reply:
Thank you !